testdata/integration_tests/SCHEDULE/SCHEDULE_WECON

```diff
@@ -0,0 +1,44 @@
+START
+  10 MAI 2007   /
```

Comment:
`testdata/integration_tests/SCHEDULE/` there are lot of decks using `MAI`. I am not sure how Eclipse and OPM handle this. From the manual, it should be `MAY`.